app/models/devise_token_auth/concerns/user.rb

```diff
@@ -158,7 +158,7 @@ def create_new_auth_token(client = nil)
     token = create_token(
       client: client,
       last_token: tokens.fetch(client, {})['token'],
-      updated_at: now
+      updated_at: now.to_s(:rfc822)
```

Comment:
wouldn't this break other's people defaults?

Reply:
@MaicolBen , Should it not be used like updated_at.to_time. Even in the whole code of this particular gem, whenever it needs to be compared with time, to_time was always used.